src/components/pages/add_edit/AddEditTextData.tsx

```
@@ -912,25 +685,53 @@ function SortableGestureItem({
   };
+  const [notToClearGesturesOrder, setNotToClearGesturesOrder] = useAtom(
+    not_to_clear_gestures_order_atom
+  );
+
+  const clearGestureVisualization = () => {
+    // Clear animated gesture lines from state
+    setAnimatedGestureLines([]);
+  };
+
+  const deleteGesture = (gestureOrder: number) => {
+    const { attributes, listeners, setNodeRef, transform, transition, isDragging } = useSortable({
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)